From: Jan Engelhardt <jengelh@inai.de>
Date: 2020-04-18 20:10:34.215071041 +0200

external.so references wx functions (including those present in base), but does
not link them => add wx libs to the link.

[   12s] /usr/include/wx-3.0/wx/filename.h:139: undefined reference to `wxFileName::Assign(wxString const&, wxPathFormat)'
[   12s] /usr/lib64/gcc/x86_64-suse-linux/9/../../../../x86_64-suse-linux/bin/ld: CMakeFiles/external.dir/email/wxEmailMessage.o: in function `wxEmailMessage::DoAddAttachment(wxString const&, wxString&) const':

Furthermore, external.so references MemChunk:: functions, but does not link
them => change to STATIC because I am too lazy.

---
 src/CMakeLists.txt          |    2 +-
 src/External/CMakeLists.txt |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

Index: SLADE-3.1.11/src/CMakeLists.txt
===================================================================
--- SLADE-3.1.11.orig/src/CMakeLists.txt
+++ SLADE-3.1.11/src/CMakeLists.txt
@@ -52,7 +52,7 @@ if (WX_GTK3)
 	set(wxWidgets_CONFIG_OPTIONS --toolkit=gtk3)
 endif (WX_GTK3)
 
-SET(WX_LIBS std aui gl stc richtext propgrid media)
+SET(WX_LIBS base std aui gl stc richtext propgrid media)
 if (NO_WEBVIEW)
 	SET(WX_LIBS ${WX_LIBS} html)
 else (NO_WEBVIEW)
Index: SLADE-3.1.11/src/External/CMakeLists.txt
===================================================================
--- SLADE-3.1.11.orig/src/External/CMakeLists.txt
+++ SLADE-3.1.11/src/External/CMakeLists.txt
@@ -30,6 +30,6 @@ file(GLOB_RECURSE EXTERNAL_SOURCES
 	${SLADE_HEADERS}
 	)
 
-add_library(external ${EXTERNAL_SOURCES})
-target_link_libraries(external ${ZLIB_LIBRARY})
+add_library(external STATIC ${EXTERNAL_SOURCES})
+target_link_libraries(external ${ZLIB_LIBRARY} ${wxWidgets_LIBRARIES})
 set(EXTERNAL_LIBRARIES external PARENT_SCOPE)
